### PR TITLE
Fix for btrfs package issue for Ubuntu distribution

### PR DESCRIPTION
**Pull request by Naresh-ibm**
The package for btrfs file system is different in ubuntu. hence it
was failing while running for btrfs file system. added the code to
fix the same issue. now it is working fine.

Signed-off-by: Naresh Bannoth <nbannoth@in.ibm.com>

### CODE REVIEW

**Comment by Naresh-ibm** [comment]
[bonnie_log2.txt](https://github.com/avocado-framework-tests/avocado-misc-tests/files/970444/bonnie_log2.txt)
[fiotest_log2.txt](https://github.com/avocado-framework-tests/avocado-misc-tests/files/970443/fiotest_log2.txt)
[fsmark_log2.txt](https://github.com/avocado-framework-tests/avocado-misc-tests/files/970445/fsmark_log2.txt)
[tiobench_log2.txt](https://github.com/avocado-framework-tests/avocado-misc-tests/files/970446/tiobench_log2.txt)

File: io/disk/bonnie.py [hunk 1]
```diff
@@ -47,6 +48,12 @@ def setUp(self):
          http://www.coker.com.au/bonnie++/experimental/bonnie++-1.03e.tgz
         """
 
+        smm = SoftwareManager()
+        if distro.detect().name == 'Ubuntu':
```
Comment (narasimhan-v) [inline io/disk/bonnie.py]: fstype check is needed

**Comment by Naresh-ibm** [comment]
[Uploading bonnie.py…]()
[Uploading bonnie_log2.txt…]()
[Uploading fiotest_log2.txt…]()
[Uploading fsmark_log2.txt…]()
[Uploading tiobench_log2.txt…]()

**Comment by Naresh-ibm** [comment]
@abdhaleegit there is a issue already opened for replacing self.skip() with self.cancel().
https://github.com/avocado-framework-tests/avocado-misc-tests/issues/449.
So once it resolves we can take a step for this.
Thanks for identifying it.

**Comment by Naresh-ibm** [comment]
amended with the changes suggested above.

**Comment by PraveenPenguin** [comment]
@abdhaleegit @narasimhan-v thanks for review

**Pull request merged**